app/helpers/handlers.js

```diff
@@ -82,6 +84,11 @@ const insertYarValue = (field, url, request) => {
     field = field.replace('[[_article_]]', replacement)
   }
 
+  if (url === 'veranda-features') {
```

Comment:
There is another method which achieves the same results here. Check the very first commit to see which is more optimal.

Reply:
As long as everything works across the app when you test it locally (via docker), then its all fine :)

Just conscious that writing code specifically for one page isn't very generalised :P

Reply:
I think the first commit version was better as it fit into the existing structure, as opposed to creating a new and specific replacement for one page only

Reply:
okay then, will use the first commit version 😃